R/parse_expr.R

```diff
@@ -36,6 +36,22 @@ parse_expr_assignment <- function(expr, src, call) {
     special <- "parameter"
   }
 
+  if (identical(special, "initial")) {
+    zero_every <- lhs$args$zero_every
+    if (!rlang::is_missing(zero_every) && !is.null(zero_every)) {
+      if (!rlang::is_integerish(zero_every)) {
+        odin_parse_error(
+          "Argument to 'zero_every' must be an integer",
+          "E1099", src, call)
+      }
+      if (!(identical(rhs$expr, 0) || identical(rhs$expr, 0L))) {
+        odin_parse_error(
+          "Initial condition of periodically zeroed variable must be 0",
+          "E1099", src, call)
```

Comment:
Do you have an uncommitted update for errors.Rmd for E1099? (OR perhaps 1099 is a placeholder - next entry would be E1019)